client/src/Components/RegisterPageComponents/Stepper/Steps/LoginInformation.jsx

```diff
@@ -0,0 +1,169 @@
+    >
+      <TextField
+        sx={{ mt: 3, width: "500px" }}
+        id="outlined-basic"
+        label="اسم النادي"
+        name="name"
+        InputProps={{
+          endAdornment: (
+            <InputAdornment position="end">
+              <PersonIcon />
+            </InputAdornment>
+          ),
+        }}
+        value={loginInformationfForm.values.name}
+        onChange={loginInformationfForm.handleChange}
+        error={!!loginInformationfForm.errors.name}
+        helperText={loginInformationfForm.errors.name}
+        variant="outlined"
+      />
+
+      <TextField
+        sx={{ mt: 3, width: "500px" }}
+        label="أدخل البريد الإلكتروني"
+        name="email"
+        InputProps={{
+          endAdornment: (
+            <InputAdornment position="end">
+              <EmailIcon />
+            </InputAdornment>
+          ),
+        }}
+        value={loginInformationfForm.values.email}
+        onChange={loginInformationfForm.handleChange}
+        error={!!loginInformationfForm.errors.email}
+        helperText={loginInformationfForm.errors.email}
+        variant="outlined"
+      />
+
+      <TextField
+        sx={{ mt: 3, width: "500px" }}
+        label="أدخل كلمة السر"
+        type={showPassword ? "text" : "password"}
+        name="password"
+        InputProps={{
+          endAdornment: (
+            <InputAdornment position="end" onClick={handleClickShowPassword}>
+              {showPassword ? (
+                <VisibilityOff sx={{ cursor: "pointer" }} />
+              ) : (
+                <Visibility sx={{ cursor: "pointer" }} />
+              )}
+            </InputAdornment>
+          ),
+        }}
+        value={loginInformationfForm.values.password}
+        onChange={loginInformationfForm.handleChange}
+        error={!!loginInformationfForm.errors.password}
+        helperText={loginInformationfForm.errors.password}
+        variant="outlined"
+      />
+      <label htmlFor="icon-button-file">
+        <Input
+          accept="image/*"
+          type="file"
+          id="icon-button-file"
+          name="image"
+          onChange={async (event) => {
+            const imageBase = await convertToBase64(
+              event.currentTarget.files[0]
+            );
+            loginInformationfForm.setFieldValue("image", imageBase);
+          }}
+        />
+        <Button
+          variant="outlined"
+          component="span"
+          sx={{
+            mt: 3,
+            height: "3.3rem",
+            width: "500px",
+            display: "flex",
+            alignItems: "center",
+            justifyContent: "start",
+          }}
+        >
+          <PhotoCamera sx={{ mr: 1 }} />
+          ادخل الشعار
+        </Button>
+        <FormHelperText id="component-error-text" error>
+          {loginInformationfForm.errors.image}
+        </FormHelperText>
+        {loginInformationfForm.values.image && (
+          <Avatar
```

Comment:
Actually when I uploaded an image, avatar show me nothing for my image, thw width is too small
So I guess maybe something like this would be better
```jsx
        {loginInformationfForm.values.image && (
          <div>
            <img
              src={loginInformationfForm.values.image}
              className="preview_img"
              alt="imageProfile"
              style={{ width: "500px" }}
            />
          </div>
        )}
        ```